### PR TITLE
FIXED::Modification of network ips from CLI

### PR DESCRIPTION
**Pull request by williamjnrkdd**
This merge request depends on: https://github.com/VultureProject/vulture-gui/pull/230
## Changed
- [ADMIN][MANAGEMENT] Changed the use of management.ip file to using rc configuration
- [ADMIN][MANAGEMENT] Added options in the CLI to change:
  - internet_ip
  - backends_outgoing_ip
  - logom_outgoing_ip
- [ADMIN][MANAGEMENT] Modified management.sh to change the above ips
- [ADMIN][MANAGEMENT] Renamed management.sh to network-ips.sh
- [NETWORK-IPS][APACHE] Changed from reloading configuration to reloading the service

### CODE REVIEW

File: home/vlt-adm/system/network-ips.sh [hunk 1]
```diff
@@ -6,11 +6,15 @@ if [ "$(/usr/bin/id -u)" != "0" ]; then
...
+internet_ip="$2"
+backends_outgoing_ip="$3"
+logom_outgoing_ip="$4"
 
-/sbin/ifconfig | grep "$ip" > /dev/null
+/sbin/ifconfig | grep "$management_ip" > /dev/null
 if [ "$?" == 0 ]; then
-    /bin/echo "$ip" > /usr/local/etc/management.ip
+    # update node network ips in /etc/rc.conf.d/network
+    /usr/sbin/sysrc -f /etc/rc.conf.d/network management_ip=$management_ip internet_ip=$internet_ip backends_outgoing_ip=$backends_outgoing_ip logom_outgoing_ip=$logom_outgoing_ip
```
Comment (frikilax) [inline home/vlt-adm/system/network-ips.sh:17]: Condition is not in sync with what needs to be modified: the user might only modify `backends_outgoing_ip` and parameters in the system won't be changed
This needs to be adapted (and separate configuration files be regenerated, especially pf.conf)

File: home/vlt-adm/system/network-ips.sh [hunk 2]
```diff
@@ -44,10 +48,10 @@ if [ "$?" == 0 ]; then
...
-        # update node management ip in Mongo
-        /home/vlt-os/env/bin/python /home/vlt-os/vulture_os/manage.py shell -c "from system.cluster.models import Node ; n = Node.objects.get(name=\"`hostname`\") ; n.management_ip = \"$ip\" ; n.save()"
-        # update management ip in apache conf
-        /home/vlt-os/env/bin/python /home/vlt-os/vulture_os/manage.py shell -c "from services.apache.apache import reload_conf ; import logging ; logger=logging.getLogger('services') ; reload_conf(logger)"
+        # update node network ips in Mongo
+        /home/vlt-os/env/bin/python /home/vlt-os/vulture_os/manage.py shell -c "from system.cluster.models import Node ; n = Node.objects.get(name=\"`hostname`\") ; n.management_ip = \"$management_ip\" ; n.internet_ip = \"$internet_ip\" ; n.backends_outgoing_ip = \"$backends_outgoing_ip\" ; n.logom_outgoing_ip = \"$logom_outgoing_ip\" ; n.save()"
+        # reload apache service
+        /home/vlt-os/env/bin/python /home/vlt-os/vulture_os/manage.py shell -c "from services.apache.apache import reload_service ; import logging ; logger=logging.getLogger('services') ; reload_service(logger)"
```
Comment (frikilax) [inline home/vlt-adm/system/network-ips.sh:54]: pf.conf also needs to be updated, and `/usr/local/bin/pfctl-init.sh` (line 43) should not be called anymore (we don't want to reset all existing rules!)

File: usr/local/bin/pfctl-get_management.sh [hunk 3]
```diff
@@ -1,10 +1,7 @@
...
 
 #This script automatically create an IP management address
-/usr/bin/touch /usr/local/etc/management.ip
-/usr/sbin/chown root:wheel /usr/local/etc/management.ip
-/bin/chmod 644 /usr/local/etc/management.ip
 ip="$(/sbin/ifconfig | /usr/bin/grep inet | /usr/bin/grep -v '127.0.0.1' \
       | /usr/bin/grep -v ' ::1 ' | /usr/bin/grep -v 'fd00::20' | /usr/bin/grep -v 'fe80:' \
       | /usr/bin/awk '{print $2}' | /usr/bin/awk -vRS="" -vOFS=' ' '$1=$1')"
-/bin/echo "$ip" | /usr/bin/sed -e 's/ .*//' > /usr/local/etc/management.ip
+/usr/sbin/sysrc -f /etc/rc.conf.d/network management_ip="$(/bin/echo "$ip" | /usr/bin/sed -e 's/ .*//')"
```
Comment (frikilax) [inline usr/local/bin/pfctl-get_management.sh:7]: the `sed`on line 7 could be done on line 5, along other filterings
that could be good to also set a test to avoid setting an empty value to the management_ip rc.conf variable

File: usr/local/bin/pfctl-init.sh [hunk 4]
```diff
@@ -1,9 +1,9 @@
 #!/bin/sh
 
 #This script restores a default configuration for PF
-management_ip="$(/bin/cat /usr/local/etc/management.ip)"
+management_ip="$(/usr/sbin/sysrc -f /etc/rc.conf.d/network -n management_ip 2> /dev/null)"
 
-grep ':' /usr/local/etc/management.ip > /dev/null
+/usr/sbin/sysrc -f /etc/rc.conf.d/network -n management_ip | /usr/bin/grep ":"  > /dev/null 2>&1
```
Comment (frikilax) [inline usr/local/bin/pfctl-init.sh:6]: missing validation that the value is not empty
